features/postman_tutorial.rst

```diff
@@ -0,0 +1,84 @@
+.. _postman_tutorial:
+
+Postman Tutorial
+================
+
+This article will walk through setting up Postman to access Firely Server restful endpoint.
+
+Prerequisites
+-------------
+Sign up for and install Postman. For more information about Postman, see `Get Started with Postman <https://www.getpostman.com/>`_.
+
+
+Using Postman
+-------------
+
+#. Click the following "Run in Postman" link:
+
+    .. raw:: html
+
+        <div align="center" class="postman-run-button"
+        data-postman-action="collection/fork"
+        data-postman-var-1="24489118-d02c5cf2-8890-4d1b-a2d5-9c7688d56793"
+        data-postman-collection-url="entityId=24489118-d02c5cf2-8890-4d1b-a2d5-9c7688d56793&entityType=collection&workspaceId=822b68d8-7e7d-4b09-b8f1-68362070f0bd"></div>
+        <script type="text/javascript">
+          (function (p,o,s,t,m,a,n) {
+            !p[s] && (p[s] = function () { (p[t] || (p[t] = [])).push(arguments); });
+            !o.getElementById(s+t) && o.getElementsByTagName("head")[0].appendChild((
+              (n = o.createElement("script")),
+              (n.id = s+t), (n.async = 1), (n.src = m), n
```

Comment:
did you try with the markdown snippet as documented at: https://learning.postman.com/docs/publishing-your-api/run-in-postman/creating-run-button/#sample-markdown-snippet

Reply:
I did but could not make it work. I copied this from other postman collections we present on other doc pages.